### ISSUE TITLE
[FEATURE REQUEST] Search results filter for Profile Features Limited / Steam is Learning / Low Confidence Metric / PlusZero

### ISSUE THREAD

**Issue by Revadike** [tfedor/AugmentedSteam#1181]
### Current Behavior (if any)
Steam search results always includes limited games (that don't give +1 to your profile game count).

### Desired Behavior
A toggle that can filter them out.

### Motivation / Use Case for Changing the Behavior
People (especially game collections) want to avoid them. They are usually lower quality games too.

**Comment by Revadike** [comment]
Inspired by https://gist.github.com/Luckz/8b13814ac60105ee2249cf16dd4eb126/raw/f4cb3d42d9b2515710acae925aa2481754142a5f/SteamSearchPlusOneFilter.user.js by @luckz

**Comment by candela97** [comment]
Is there a way to determine this status from the search row html? Apart from the label on the store page.

**Comment by Revadike** [comment]
> Is there a way to determine this status from the search row html? Apart from the label on the store page.

Seems like it. Check out the solution from the linked script.

**Comment by MxtOUT** [comment]
It seems to fetch the [games that are profile limited](https://bartervg.com/browse/tag/481/json/) from barter.vg

**Comment by Revadike** [comment]
Oh, you're right. Sorry, I overlooked that.

**Comment by candela97** [comment]
Yeah I checked the tag 481 but it seems to be Barter's own tag id rather than Steam's. I wonder how they maintain the list of apps?

**Comment by Revadike** [comment]
@bartervg
